pyCGM_Single/pycgmIO.py

```diff
@@ -59,16 +59,62 @@
 EA=SA+72*3
 
 def createMotionDataDict(labels,data):
-	motiondata = []
-	for frame in data:
-		mydict={}
-		for label,xyz in zip(labels,frame):
-			l=str(label.rstrip())
-			mydict[l] = xyz
-		motiondata.append(mydict)
-	return motiondata
+    """Creates an array of motion capture data given labels and data
+
+    Arguments
+    ---------
+    labels : array
+        List of marker position names. Example:
+        ['LFHD', 'RFHD', 'LBHD', ...]
+    data : array
+        List of xyz coordinate positions corresponding to the marker
+        names in `labels`. Indices of `data` correspond to frames in the trial.
+        Example for two frames of trial: 
+            [[[x1, y1, z1], [x2, y2, z2], ...]], 
+             [[x1, y1, z1], [x2, y2, z2], ...]]]
+
+    Returns
+    -------
+    motiondata : array
+        List of dict. Indices of `motiondata` correspond to frames
+        in the trial. Keys in the dictionary are marker names and 
+        values are xyz coordinates of the corresponding marker.
+        Example for two frames of trial:
+            [{marker1_name : [x1, y1, z1], marker2_name: [x2, y2, z2], ...}
+             {marker1_name : [x1, y1, z1], marker2_name: [x2, y2, z2], ...}]
+
+    """
+    motiondata = []
```

Comment:
please fix this commit change. It should just be addition of the docstring, not the removal and replacement of the code so its easier to track modifications.